python/tank/pipelineconfig.py

```diff
@@ -865,31 +931,37 @@ def get_core_hooks_location(self):
         
         :returns: path string
         """
-        return os.path.join(self._pc_root, "config", "core", "hooks")
+        return os.path.join(
+            self._get_config_core_folder(),
+            "hooks"
+        )
 
     def get_schema_config_location(self):
         """
         Returns the location of the folder schema
         
         :returns: path string
         """
-        return os.path.join(self._pc_root, "config", "core", "schema")
+        return os.path.join(
+            self._get_config_core_folder(),
+            "schema"
+        )
 
     def get_config_location(self):
         """
-        Returns the config folder for the project
-        
+        Returns the config folder location for the project
+
         :returns: path string
         """
-        return os.path.join(self._pc_root, "config")
+        return self._descriptor.get_config_folder()
```

Comment:
you are calling a method that just calls another method - could this be simplified?